internal/prompter/prompter.go

```diff
@@ -72,15 +72,26 @@ func (p *surveyPrompter) Select(message, defaultValue string, options []string)
 	return
 }
 
-func (p *surveyPrompter) MultiSelect(message, defaultValue string, options []string) (result []string, err error) {
+func (p *surveyPrompter) MultiSelect(message string, defaultValue, options []string) (result []string, err error) {
 	q := &survey.MultiSelect{
 		Message:  message,
 		Options:  options,
 		PageSize: 20,
 	}
 
-	if defaultValue != "" {
-		q.Default = defaultValue
+	// TODO will I regret this returning []string and not []int?
+
+	if len(defaultValue) > 0 {
+		// TODO I don't actually know that this is needed, just being extra cautious
```

Comment:
Should this actually thrown an error when a default value is specified that is not in the options list? Feels like raising an error there would be appropriate and possibly find typo bugs.

Reply:
As the comment in `Select` indicates there are situations where a default value might not be in the option list, I am curious as to what situations that occurs in.

Reply:
I'm also curious. I can't remember if it actually came up or if I was just trying to avoid introducing errors into a release.